### PR TITLE
Small language fixes

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: src/tray.cpp [hunk 1]
```diff
@@ -200,7 +200,7 @@ void TrayPrivate::notify(QModelIndex const & index, bool removing)
     {
         mConnectionsToNotify.removeOne(index);
         summary = Tray::tr("Connection lost");
-        body = Tray::tr("We have just lost the connection to %1 '%2'.");
+        body = Tray::tr("Lost the connection to %1 '%2'.");
```
Comment (palinek) [inline src/tray.cpp]: I'm not a native English speaker... but the body should be a "normal" sentence not just a "subject/heading"...

File: src/tray.cpp [hunk 2]
```diff
@@ -251,13 +251,13 @@ Tray::Tray(QObject *parent/* = nullptr*/)
...
     d->mActEnableNetwork = d->mContextMenu.addAction(Tray::tr("Enable Networking"));
-    d->mActEnableWifi = d->mContextMenu.addAction(Tray::tr("Enable Wi-fi"));
+    d->mActEnableWifi = d->mContextMenu.addAction(Tray::tr("Enable Wi-Fi"));
     d->mContextMenu.addSeparator();
     QAction * enable_notifications = d->mContextMenu.addAction(Tray::tr("Enable notifications"));
     d->mContextMenu.addSeparator();
     d->mActConnInfo = d->mContextMenu.addAction(QIcon::fromTheme(QStringLiteral("dialog-information")), Tray::tr("Connection information"));
     d->mActDebugInfo = d->mContextMenu.addAction(QIcon::fromTheme(QStringLiteral("dialog-information")), Tray::tr("Debug information"));
-    connect(d->mContextMenu.addAction(QIcon::fromTheme(QStringLiteral("document-edit")), Tray::tr("Edit connections...")), &QAction::triggered
+    connect(d->mContextMenu.addAction(QIcon::fromTheme(QStringLiteral("document-edit")), Tray::tr("Edit connections…")), &QAction::triggered
```
Comment (palinek) [inline src/tray.cpp]: I know, that we're living in 21st century, but I'm oldfashioned in this regard and don't want to have non-ASCII characters in the source code.

**Comment by palinek** [comment]
BTW... thanks for your translation contribution, you're the first one who used the yesterday deployed weblate for nm-tray.

File: src/tray.cpp (hunk 1 above)
Comment (comradekingu) [reply src/tray.cpp]: Looks fine to me, it is no less a complete sentence without the redundant "we have just".

**Comment by comradekingu** [comment]
@palinek :) The links to the source code aren't set up correctly. The idea is to be able to click the source directory and get to, in this case, GitHub directly.

**Comment by palinek** [comment]
> The links to the source code aren't set up correctly. The idea is to be able to click the source directory and get to, in this case, GitHub directly.

It's fixed now....I believe.

File: src/tray.cpp (hunk 1 above)
Comment (palinek) [reply src/tray.cpp]: OK... but, please, keep the style consistent, "Lost **the** connection..." vs. "Established connection..."

**Comment by comradekingu** [comment]
I remember it working. Hosted Weblate down atm.
Reworked the offending strings.

**Comment by palinek** [comment]
Landed in a3506b3c, thanks

**Pull request closed without merging**